### PR TITLE
Document the new property path options of the Range constraint

### PR DESCRIPTION
**Pull request by Lctrs**
Related PR : https://github.com/symfony/symfony/pull/31511

### CODE REVIEW

File: reference/constraints/Range.rst [hunk 1]
```diff
@@ -383,6 +407,28 @@ Parameter        Description
...
+It defines the object property whose value is used as ``min`` option.
+
+For example, if you want to compare the ``$endDate`` property of some object
+with regard to the ``$startDate`` property of the same object, use
+``minPropertyPath="startDate"`` in the range constraint of ``$endDate``.
+
+.. tip::
+
+    When using this option, its value is available in error messages as the
+    ``{{ limit_path }}`` placeholder. Although it's not intended to
```
Comment (OskarStark) [inline reference/constraints/Range.rst]: Is this true? I mean limit_path for min and max? That mean I cannot use both combined?

Comment (Lctrs) [reply reference/constraints/Range.rst]: I agree this can be improved. Will update the corresponding PR.

Comment (OskarStark) [reply reference/constraints/Range.rst]: Thanks, please add tests for using both in combination too 👌🏻

Comment (Lctrs) [reply reference/constraints/Range.rst]: Already done. :)

**Comment by OskarStark** [comment]
Please target 4.4 brach instead of master

**Comment by Lctrs** [comment]
@OskarStark Ref. PR has been merged. :)

**Comment by javiereguiluz** [comment]
@Lctrs what a nice contribution! And you even included the "versionadded" directives too, so I can only say ... thanks a lot!

**Pull request merged**